src/styles/index.scss

```diff
@@ -2,2 +1,87 @@
+/* Глобальные настройки */
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  background-color: #f9f9f9;
+}
+
+/* Карточки продуктов */
+.product-cards {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  padding: 20px;
+
+  &__item {
+    width: 200px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+
+    &[data-qa='card'] {
+      border-color: #007bff;
+      animation: highlight 0.5s ease; /* Анимация, которая отключается в тестах */
+    }
+  }
+
+  &__image {
+    width: 100%;
+    height: auto;
+    border-radius: 8px 8px 0 0;
+  }
+
+  &__title {
+    font-size: 18px;
+    font-weight: 700;
+    margin: 10px 0;
+  }
+
+  &__price {
+    font-size: 16px;
+    color: #333;
+    margin: 10px 0;
+  }
+
+  &__buy {
+    display: inline-block;
+    text-decoration: none;
+    color: #fff;
+    background-color: #007bff;
+    padding: 10px 20px;
+    border-radius: 4px;
+    font-weight: 700;
+    transition: background-color 0.3s;
+    &:hover {
+      background-color: #0056b3;
```

Comment:
Consider using SCSS variables for colors like '#0056b3' to maintain consistency and make future changes easier. Place these variables in a separate file as per the task requirements.